### PR TITLE
Add validateMaximum & validateMinimum

### PR DESCRIPTION
**Pull request by heldersepu**
This address some of the validation requested on Issue #993

### CODE REVIEW

File: src/core/utils.js [hunk 1]
```diff
@@ -539,6 +553,16 @@ export const validateParam = (param, isXml) => {
       errors.push("Required field is not provided")
       return errors
     }
+    
+    if (maximum) {
+      let err = validateMaximum(value, maximum)
+      if (err) errors.push(err)
+    }
+
+    if (minimum) {
```
Comment (owenconti) [inline src/core/utils.js]: Will this handle negative minimum values?

Comment (heldersepu) [reply src/core/utils.js]: Sure it will, what make you think it won't?

Comment (owenconti) [reply src/core/utils.js]: I thought for a second that negative integers would evaluate as false, but I was wrong. The only number that does is `0`.

Comment (owenconti) [reply src/core/utils.js]: Thanks for adding the test to verify!

**Comment by shockey** [comment]
Thanks @heldersepu!

**Pull request merged**